src/transfer/transfer.repository.ts

```diff
@@ -11,38 +11,71 @@ export class TransferRepository {
 
   static async getMany(filter: any, pageNum = 0, pageSize = 0): Promise<IPaginatedTransfer[]> {
     const startingIndex : number = pageNum * pageSize;
-    const aggregationQuery : any[]= [
-      { 
+    const aggregationQuery : any[] = [
+      {
         $match: filter
       },
-      { 
+      {
         $group: {
           _id: '$reqID',
           docs:{
             $first: {
-                "_id": "$_id",
-                "reqID": "$reqID",
-                "fileID": "$fileID",
-                "fileName": "$fileName",
-                "classification": "$classification",
-                "fileOwnerID": "$fileOwnerID",
-                "status": "$status",
-                "userID": "$userID",
-                "sharerID": "$sharerID",
-                "createdAt": "$createdAt",
-                "destination": "$destination"
-              }
+              _id: '$_id',
+              reqID: '$reqID',
+              fileID: '$fileID',
+              fileName: '$fileName',
+              classification: '$classification',
+              fileOwnerID: '$fileOwnerID',
+              status: '$status',
+              userID: '$userID',
+              sharerID: '$sharerID',
+              createdAt: '$createdAt',
+              destination: '$destination',
             }
           }
+        }
       },
-    { $sort: { _id: -1 }}];
-    if(pageSize > 0) {
-      aggregationQuery.push({ $skip: startingIndex }, { $limit: pageSize })
+    { $sort: { _id: -1 } }];
+    if (pageSize > 0) {
+      aggregationQuery.push({ $skip: startingIndex }, { $limit: pageSize });
     }
     const transfers: IPaginatedTransfer[] = await transferModel.aggregate(aggregationQuery);
     return transfers;
   }
 
+  static async getSize(filter: any): Promise<number> {
+    const aggregationQuery : any[] = [
+      {
+        $match: filter
+      },
+      {
+        $group: {
+          _id: '$reqID',
+          docs:{
```

Comment:
there is no need to specify the "docs", you only want to return the amount of documents